src/content/advanced/iam-roles-for-service-accounts/index.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: IAM Roles for Service Accounts
+title: AWS IAM Roles for Service Accounts 
```

Comment:
Converting to sentence case

```suggestion
linkTitle: IAM roles for service accounts
title: AWS IAM roles for service accounts 
```

Regarding the title: could it be more specific, indicating what use case we are addressing here?